src/main/archetype/README.md

```diff
@@ -42,6 +42,10 @@ Or to deploy only a single content package, run in the sub-module directory (i.e
 
     mvn clean install -PautoInstallPackage
 
+#if ( $precompiledScripts == "y")
+## Precompiled Scripts
```

Comment:
I believe this is missing a reference to `README-precompiled-scripts.md`

Reply:
No, I forgot to remove this section. Initially I thought these files would be filtered too, but since they aren't I created the `README-precompiled-scripts.md` and forgot to remove this. I will remove this section in the next commit.